### PR TITLE
Feat(coinmarket): edit quote payment methods to accept even not speci…

### LINKED ISSUE #9228: Trezor Suite: Flexible display of payment methods
To be able to add new payment methods without waiting Trezor Suite new release.

If there is missing translation, show fallback returned from Invity API.

Let’s remove all specific payment methods from Trezor Suite and keep only bank card and bank transfer.

Also need to update Invity API server payment method names so they are with capital case.

Dev testing:

Try to rename ideal to ideal2 in current develop and then in your branch

Please post here screenshot of new/renamed payment method in “old“ Trezor Suite.

### PR DESCRIPTION
**Pull request by zuzjes**
## Description
The goal is to be able add even not yet specified payments methods for sell, buy and savings.

## Related Issue

Resolve #9228 

## Screenshots:
Screenshot 2023-08-11 at 13 00 43
 - Example of new payment. The payment method icon is not visible because I add non existing payment method - with no image.
 -

### CODE REVIEW

**Comment by matejkriz** [comment]
Does this have an related github issue?

**Comment by IvaFingerova** [comment]
here is a general one https://github.com/trezor/trezor-suite/issues/9228 and here is a connected issue by Vlad from the past https://github.com/satoshilabs/invity-api/pull/193

File: packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx [hunk 1]
```diff
@@ -35,34 +35,45 @@ const Text = styled.div`
 
 interface CoinmarketPaymentTypeProps {
     children?: React.ReactNode;
-    method?: BuyCryptoPaymentMethod | SavingsPaymentMethod;
+    method?: BuyCryptoPaymentMethod | SellCryptoPaymentMethod | SavingsPaymentMethod;
+    methodName?: string | undefined;
```
Comment (matejkriz) [inline packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx]: nitpick, `?` and `| undefined` is the same
```suggestion
    methodName?: string;
```

File: packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx [hunk 2]
```diff
@@ -35,34 +35,45 @@ const Text = styled.div`
...
+    children,
+    method,
+    methodName,
+}: CoinmarketPaymentTypeProps) => (
     <Wrapper>
-        {!method && (
-            <Text>
-                <Translation id="TR_PAYMENT_METHOD_UNKNOWN" />
-            </Text>
-        )}
```
Comment (matejkriz) [inline packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx:47]: unknown fallback no more? looks safer to me to have it

**Review by matejkriz — APPROVED** [review]
LGTM, just two nitpicks – feel free to mark them as resolved

File: packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx (hunk 1 above)
Comment (zuzjes) [reply packages/suite/src/components/wallet/CoinmarketPaymentType/index.tsx]: so my change is cleaner or not? At least shorter?

**Pull request merged**